### PR TITLE
test: BSS+ signature suite interop tests (with Mattr)

### LINKED ISSUE #2223: BSS+ signature suite interop tests (with Mattr)
https://github.com/mattrglobal/jsonld-signatures-bbs

### PR DESCRIPTION
**Pull request by kdimak**
In this PR, a check of `BbsBlsSignature2020` interoperability between aries-framework-go and reference implementation from [Mattr](https://github.com/mattrglobal/jsonld-signatures-bbs) is made.
Input VC document and BBS+ key pair are taken from the [sample](https://github.com/mattrglobal/jsonld-signatures-bbs/tree/master/sample).

`verifiable` package was used directly through WASM. We can later change this to use `aries-js-worker` as soon as we support BBS+ in KMS (#2295).

closes #2223

Signed-off-by: Dmitriy Kinoshenko <dkinoshenko@gmail.com>

### CODE REVIEW

**Review by Baha-sk — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -72,6 +72,10 @@ bdd-test-js-local:
 vc-test-suite: clean
 	@scripts/run_vc_test_suite.sh
 
+.PHONY: bss-interop-test
+bss-interop-test:
```
Comment (fqutishat) [inline Makefile:76]: can you create CI build for this

Comment (fqutishat) [reply Makefile:76]: we need to make sure it's passing in CI

Comment (fqutishat) [reply Makefile:76]: like test-suite / VC test suite

Comment (kdimak) [reply Makefile]: Added bbs-interop / BBS+ interop test.
Thanks @fqutishat for the proposal.

**Review by fqutishat — APPROVED** [review]

**Review by llorllale — APPROVED** [review]

**Review by kdenhartog — COMMENTED** [review]
Just got reviewed. Looks good to me. I'll file an issue once we've cut the new release with the breaking changes around the security v3-unstable context.

**Pull request merged**